src/languages/php.js

```diff
@@ -156,11 +156,11 @@ export default function(hljs) {
       },
       {
         className: 'class',
-        beginKeywords: 'class interface',
+        beginKeywords: 'class interface trait enum',
```

Comment:
Try using `variants` here, something like:

```
variants: [
  { beginKeywords: "enum", illegal: [reduced set] },
  { beginKeywordS: "class interface trait", illegal: [full set] }
]
```

It's a shortcut to writing all new rules (which is what the mode compiler does internally).. All the other keys (from the top) are inherited.